spec/system/user/view_user_index_spec.rb

```diff
@@ -113,4 +117,12 @@ def shows_comments
       end
     end
   end
+
+  def shows_last_comments
+    stub_const("CommentsHelper::MAX_NBR_OF_COMMENTS_TO_RENDER", 1)
+    visit "/#{user.username}"
+    within("#substories .profile-comment-card .pt-3 .fs-base") do
+      expect(page).to have_content("View last 1 Comments")
```

Comment:
**suggestion (non-blocking):** maybe we can use the `pluralize` method in the views and then update this spec to read:
```ruby
expect(page).to have_content("View last 1 Comment")
```

Reply:
I do concur. It will be better with pluralize.